api/index.ts

```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+import swaggerUi from "swagger-ui-express";
+import express from 'express';
+import http from 'http';
+
+const { PORT } = process.env;
+
+const openApiDocumentation = JSON.parse(
+  fs.readFileSync("./apis.json").toString()
+);
+
+console.log("Test", openApiDocumentation);
+const app = express();
+app.get("/docs", swaggerUi.serve, swaggerUi.setup(openApiDocumentation));
```

Comment:
This should be app.use, 
think that is the problem

Reply:
@amarts

Reply:
I tried both, and still seeing the issue.

Reply:
I got it when i tried

Reply:
Ack. You are right! using `app.use()` instead of app.get() it is working. Sending the fix.